### ISSUE TITLE
Plan to translate into Chinese.

### ISSUE THREAD

**Issue by ZhangLanDan** [theelous3/asks#126]
Asks is a fantastic async lib for a novice like me for it looks familiar with Requests.  I cannot help thinking sharing it with all the  Chinese learner so I was wondering if I could translate the docs into Chinese.
( First time to write an issue)

**Comment by theelous3** [comment]
Absolutely. I love that. If there is anything I can do to help, feel free to ask :)

**Comment by jean** [comment]
@ZhangLanDan `asks` uses Sphinx for docs, if you like you can follow the [Sphinx internationalization guide](http://www.sphinx-doc.org/en/master/usage/advanced/intl.html) to get tool support for translation. 

For example you can set up a project in transifex and crowdsource the translation. (Note: "_We offer Transifex for free to Open Source projects that have no funding, revenue, and/or commercialization model._")
